src/pyspiffe/bundle/jwt_bundle/jwt_bundle.py

```diff
@@ -51,7 +64,5 @@ def find_jwt_authority(self, key_id: str) -> _PUBLIC_KEY_TYPES:
         if not key_id:
             raise ValueError('key_id cannot be empty.')
 
-        key = self._jwt_authorities.get(key_id, None)
-        if not key:
-            raise AuthorityNotFoundError(key_id)
-        return key
+        with self.lock:
+            return self._jwt_authorities.get(key_id)
```

Comment:
It does not raise AuthorityNotFoundError anymore, it only raises ValueError. need to fix docstring.

Reply:
Good catch Glaucimar. It is fixed now. I also added a note to indicate that the method returns None if the key_id is not found.